gui/default/assets/css/overrides.css

```diff
@@ -401,6 +372,15 @@ ul.three-columns li, ul.two-columns li {
     width: 100%;
 }
 
+/*
+ * Specific tweaks for folderTree component
+ */
+
+/* Maintain consistent padding with the restoreTree component */
+#folderTree td {
+    padding: 2px 0;
+}
```

Comment:
Maybe move all tree-related code to `tree.css`? This should help with future maintainability.